lib/matplotlib/offsetbox.py

```diff
@@ -44,22 +44,18 @@ def bbox_artist(*args, **kwargs):
     if DEBUG:
         mbbox_artist(*args, **kwargs)
 
-# _get_packed_offsets() and _get_aligned_offsets() are coded assuming
-# that we are packing boxes horizontally. But same function will be
-# used with vertical packing.
-
 
 def _get_packed_offsets(wd_list, total, sep, mode="fixed"):
-    """
-    Given a list of (width, xdescent) of each boxes, calculate the
-    total width and the x-offset positions of each items according to
-    *mode*. xdescent is analogous to the usual descent, but along the
-    x-direction. xdescent values are currently ignored.
+    r"""
+    Horizontally pack boxes specified by their ``(width, xdescent)`` pair.
```

Comment:
I don't think we should use *horizontal* before the disclaimer on direction.
```suggestion
    Distibute boxes within a given *total* space.
```

Reply:
yes